lib/gauge_loop_trace.cu

```diff
@@ -13,6 +13,7 @@ namespace quda {
     std::vector<reduce_t>& loop_traces;
     double factor;
     const paths<1> p;
+    unsigned int sharedBytesPerThread() const override { return 4 * sizeof(int); } // for threda_array
```

Comment:
`thread` not `threda`

Reply:
13075c7bd787358cfda507de65cfb588ce5b435b